deploy/chart/templates/0000_90_olm_01-prometheus-rule.yaml

```diff
@@ -9,12 +9,23 @@ metadata:
     role: alert-rules
 spec:
   groups:
-    - name: olm.failing_operators.rules
+    - name: olm.csv_abnormal.rules
       rules:
-        - alert: FailingOperator
+        - alert: CsvAbnormalReplacingOver30Min
+          expr: csv_abnormal{phase=~"(^Replacing$|^Failed$|^Unknown$)",exported_namespace=~"(^openshift.*|^kube.*|^redhat.*|^default$)"}
+          for: 30m
```

Comment:
Do we not think it important to get an alert for a `Failing` state before 30m have passed?

I would assume a shorter time for `Failing` than for `Replacing`/`Unknown`. I also think `Pending` and `Deleting` would be reasonable things to alert on if they're in that state for > 30m.

Reply:
@ecordell Thanks! I agree, using a shorter time for `Failing` sounds better. Umm, but I am not sure what should it be, what about 15m?

Reply:
I suggest it should be much much shorter -- [1, 2] minutes maybe -- to pick up on flapping CSVs; i.e. CSVs that cycle in and out of `Failing` rapidly. @operator-framework/team-olm thoughts?

Reply:
@njhale Hello, what about using `5m` for `Failed` phase, and using `30m` for `Replacing/Pending/Deleting/Unknown`?